tests/test_servo.py

```diff
@@ -34,7 +34,7 @@ def create_monitoring(connect_to_slave, pytestconfig):
     servo, net = connect_to_slave
     servo.monitoring_disable()
     servo.monitoring_remove_all_mapped_registers()
-    registers_key = ["CL_POS_SET_POINT_VALUE"]
+    registers_key = ["CL_CUR_D_REF_VALUE"]
```

Comment:
Why is this change needed?

Reply:
Beacuse CL_POS_SET_POINT_VALUE is CYCLIC_RX so it could not be mapped in monitoring. Not sure why it worked in Capitan.